code/modules/reqs/supply.dm

```diff
@@ -302,12 +282,14 @@ GLOBAL_LIST_EMPTY_TYPED(exports_types, /datum/supply_export)
 	.["supplypackscontents"] = SSpoints.supply_packs_contents
 	.["elevator_size"] = SSshuttle.supply?.return_number_of_turfs()
 
-/datum/supply_ui/ui_data(mob/user)
+/datum/supply_ui/ui_data(mob/living/user)
 	. = list()
-	.["currentpoints"] = round(SSpoints.supply_points)
+	.["currentpoints"] = round(SSpoints.supply_points[user.faction])
 	.["requests"] = list()
 	for(var/i in SSpoints.requestlist)
 		var/datum/supply_order/SO = SSpoints.requestlist[i]
+		if(SO.faction != user.faction)
+			continue
```

Comment:
this filters the supply orders. If a request was made by a rebel, then loyalists won't be able to do see it